target/linux/ath79/generic/base-files/etc/board.d/02_network

```diff
@@ -411,6 +411,7 @@ ath79_setup_interfaces()
 			"0@eth0" "2:lan:3" "3:lan:2" "4:lan:1" "5:lan:4"
 		;;
 	tplink,tl-wr842n-v2|\
+	tplink,tl-wr841hp-v2|\
```

Comment:
Alphabetic sorting.

Reply:
sorting is done. Thanks

Reply:
Every requested changes has done. Please Check if more changes are needed. Thank you for your time.